src/commands/guild/GuildShopCommand.ts

```diff
@@ -96,7 +95,7 @@ function getGuildXPShopItem(guildShopTranslations: TranslationModule) {
 	);
 }
 
-function getFoodShopItem(guildShopTranslations: TranslationModule, name: string, language: string, amounts: number[]) {
+function getFoodShopItem(guildShopTranslations: TranslationModule, name: string, language: string, amounts: number[], interaction: CommandInteraction) {
```

Comment:
coucou pieta, le bot a raison, fais moi descendre ce nombre à 4, YOU CAN DO THIS